core/src/test/java/nl/nn/adapterframework/doc/DocWriterNewIntegrationTest.java

```diff
@@ -6,33 +6,92 @@
 import java.io.IOException;
 import java.io.Writer;
 
+import javax.xml.XMLConstants;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+
+import org.apache.logging.log4j.Logger;
 import org.junit.Ignore;
 import org.junit.Test;
 
+import nl.nn.adapterframework.core.Resource;
 import nl.nn.adapterframework.doc.model.FrankDocModel;
 import nl.nn.adapterframework.doc.model.FrankElement;
 import nl.nn.adapterframework.doc.model.FrankElementStatistics;
+import nl.nn.adapterframework.util.LogUtil;
 
-@Ignore("Test takes a long time to run, and gives little information")
 public class DocWriterNewIntegrationTest {
+	private static Logger log = LogUtil.getLogger(DocWriterNewIntegrationTest.class);
+	private static final String TEST_CONFIGURATION_FILE = "testConfiguration.xml";
+
+	@Ignore("This test takes too long.")
```

Comment:
Je kan ook assumeTrue of assumeFalse gebruiken: `assumeFalse(TestAssertions.isTestRunningOnCI());`
Dan draait hij alleen lokaal (mocht je dat willen natuurlijk!!!)

Reply:
Ik denk dat we deze testen juist niet lokaal willen maar wel tijdens CI/CD. De testen duren lang, dus als je aan het ontwikkelen bent zijn deze testen storend. Maar deze testen checken niet alleen de XSD generatie code, maar ook alle overige code. We testen dat de Java classes van het FF! in een bruikbare XSD te vertalen zijn.

Reply:
Okey, het ging mij er puur om dat je bewust bent van de `TestAssertions` helper klasse en het gebruik van assumeTrue om tests conditioneel te maken :)

Reply:
Nu snap ik het. Ik heb nu gemaakt dat deze testen alleen in CI/CD draaien.